custom_components/hikvision_next/hikvision_device.py

```diff
@@ -59,8 +61,10 @@ def __init__(
         username = config[CONF_USERNAME]
         password = config[CONF_PASSWORD]
         varify_ssl = config.get(CONF_VERIFY_SSL, True)
+        force_rtsp_port = config.get(FORCE_RTSP_PORT, False)
```

Comment:
I think checkbox is not needed, empty forced port number would disable this option

Reply:
You are right. I pushed the change